### ISSUE TITLE
-X excludeColumnRegex grays out columns instead of hiding them

### ISSUE THREAD

**Issue by BrannJoly** [schemaspy/schemaspy#1528]
I'm using -X excludeColumnRegex to hide some columns in my schema.

## Expected Behavior
I want to hide those columns entirely, not gray them out

## Current Behavior
the columns are visible but grayed out, like this

<img width="198" alt="image" src="https://github.com/user-attachments/assets/41261836-c251-48f7-85b6-5fddfc42f4b5">

## Context

* Database type (e.g. MySQL):psql
* Full command including arguments how you run SchemaSpy: `      -X ".*(event_id|pagination_token)$"`

## Your Environment
* Version 6.2.4

**Comment by npetzall** [comment]
Hmm, would be easy to add, but need a new cli option.

They are shown and greyed since they are either indexed or considered foreign keys. Which is sort of the feature, remove a bunch of lines from de graph. But they are still important. Which is why they are shown.

So the option to hide them would make the model easier to see, but it would hide important "system" dependencies.

**Comment by BrannJoly** [comment]
> Hmm, would be easy to add, but need a new cli option.
> 
> They are shown and greyed since they are either indexed or considered foreign keys. Which is sort of the feature, remove a bunch of lines from de graph. But they are still important. Which is why they are shown.
> 
> So the option to hide them would make the model easier to see, but it would hide important "system" dependencies.

I understand the rational behind keeping them, but in my case it's just noise. I have a huge data model, every single table has a pagination_token, and FK to event_ids, and these are 4 lines of pure noise added across hundreds of tables :/

Still, I'm very happy with the result, way better to onboard new devs than showing them a raw SQL schema!
Thank you for your work
Brann

**Comment by npetzall** [comment]
Well, the rational is more or less for not changing the current behaviour, sort of why it would be a new argument.

**Comment by BrannJoly** [comment]
> Well, the rational is more or less for not changing the current behaviour, sort of why it would be a new argument.

Yes I undestand. I think the doc could clearly be updated/improved
and maybe add an option -neverHideFkAndIndexedColumns defaulted to true, or something along those lines!
